Week3/homework/step2-3.js

```diff
@@ -4,40 +4,57 @@
 function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  for (let i = 0; i < num; i++) {
+    if (num < 0) {
```

Comment:
I think this `if` condition belongs outside the `for` loop rather than inside. Could you please check?

Reply:
Okay I did it thanks!